src/components/NetworkTable/constants.ts

```diff
@@ -14,7 +14,7 @@ export const NETWORK_DEFAULT_NODES_COLUMNS: NodesColumnId[] = [
     'ClockSkew',
 ];
 
-export const NETWORK_REQUIRED_NODES_COLUMNS: NodesColumnId[] = ['NodeId'];
+export const NETWORK_REQUIRED_NODES_COLUMNS: NodesColumnId[] = ['NodeId', 'NetworkHost'];
```

Comment:
`NetworkTable` default columns do not require `SystemState` field from backend, so without `NetworkHost` node will be displayed as unavailable